src/arch/aarch64/smc.c

```diff
@@ -32,6 +33,29 @@ typedef enum {
     SMC_CALL_RESERVED = 64,
 } smc_call_id_t;
 
+#if defined(CONFIG_ALLOW_SMC_CALLS)
+/* Service handlers for services that require "physical" access to EL3 components
+   (vs emulation) are listed here.
+
+   'Active' handler is a pointer to fucntion that actually will be invoked;
+   by default active handlers are set to NULL. Application should assign to an active handler
+   wheither a default handler that unconditionally forwards calls to Secure Monitor:
+
+   extern handle_service_type handle_xxx;
+   handle_service_type handle_xxx = default_handle_service;
```

Comment:
In my opinion the user of the library should not have to do this, and instead just have a wrapper for setting the handler.

Reply:
Also thank you for writing comments for how to use the API.

Reply:
> In my opinion the user of the library should not have to do this, and instead just have a wrapper for setting the handler.

Agreed, that would be easier to use.

Reply:
API changed